docs/user_api/pdf/physics.rst

```diff
@@ -0,0 +1,12 @@
+
+
+.. autosummary::
+    :toctree: _generated/physics
+
+    zfit_physics.pdf.Argus
+    zfit_physics.pdf.RelativisticBreitWigner
```

Comment:
again @ikrommyd FYI: the autosummary directive creates a summary of objects that are listed, the toctree will be generated in `_generated`/physics -> `_generated` is just a folder, conventional, and physics the name of the file to be produced (which can be arbitrary, but we take the same as the rst file here).